docs/sections/user_guide/api/config.rst

```diff
@@ -0,0 +1,6 @@
+**********************
+``uwtools.api.config``
+**********************
+
+.. automodule:: uwtools.api.config
+    :members:
```

Comment:
This is the `autodoc` configuration. It states which module to parse for docstrings to turn into HTML docs in this file.